react-graph/src/Step.tsx

```diff
@@ -0,0 +1,112 @@
+// ******************************************************************************************************
+//  Step.tsx - Gbtc
+//
+//  Copyright � 2020, Grid Protection Alliance.  All Rights Reserved.
+//
+//  Licensed to the Grid Protection Alliance (GPA) under one or more contributor license agreements. See
+//  the NOTICE file distributed with this work for additional information regarding copyright ownership.
+//  The GPA licenses this file to you under the MIT License (MIT), the "License"; you may not use this
+//  file except in compliance with the License. You may obtain a copy of the License at:
+//
+//      http://opensource.org/licenses/MIT
+//
+//  Unless agreed to in writing, the subject software distributed under the License is distributed on an
+//  "AS-IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. Refer to the
+//  License for the specific language governing permissions and limitations.
+//
+//  Code Modification History:
+//  ----------------------------------------------------------------------------------------------------
+//  12/13/2022 - A. Hagemeyer
+//       Generated original version of source code.
+//
+// ******************************************************************************************************
+
+import * as React from 'react'
+
+interface IProps {
+    steps:
+    {
+        short: string,
+        long: string,
+        id: string | number
+    }[],
+    activeStep: string | number,
+    height: number,
```

Comment:
I agree with Gabriel
```suggestion
    height?: number|string,
```
and when you use `props.height` change it to `props.height == undefined? '100%' : props.height`